src/lib.rs

```diff
@@ -1324,6 +1324,11 @@ where UP: UserPresence,
         credential_key: KeyId,
     ) -> Result<Option<ctap2::get_assertion::ExtensionsOutput>> {
         if let Some(hmac_secret) = &extensions.hmac_secret {
+            if let Some(pin_protocol) = hmac_secret.pin_protocol {
```

Comment:
If I may, you can directly make the check without the assignation : `if hmac_secret.pin_protocol.is_some() && hmac_secret.pin_protocol != Some(1)`

Reply:
For me the current version is easier to read, but I can change it if you prefer a single if statement.